uninstall.sh

```diff
@@ -0,0 +1,350 @@
+#!/bin/bash
+set -u
+shopt -s extglob
+
+# Set up temp dir
+tmpdir=/tmp/uninstall.$$
+mkdir -p "$tmpdir" || Kernel_abort "Unable to create temp dir '$tmpdir'"
+trap '
+  rm -fr "$tmpdir"
+  # Invalidate sudo timestamp before exiting
+  /usr/bin/sudo -k
+' exit
```

Comment:
I think it's common to use UPPER CASE for traps.

Reply:
I've seen it both ways in about equal measure, but OK, I'll fix that.